include/Options/Options.php

```diff
@@ -146,16 +146,6 @@ public function protect_wp_option_storage( $value ) {
 	 * @return void
 	 */
 	public function init_options_for_blog( $blog_id ): void {
```

Comment:
The `$blog_id` parameter is not needed anymore, we can use `$this->current_blog_id`.

Reply:
Good idea, I've been a bit further by making the method name more explicit and make it private.
See 1a723f14.

Reply:
Good idea.